components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/SCIMCommonUtils.java

```diff
@@ -127,6 +132,37 @@ public static String getSCIMRoleV2URL(String id) {
         return StringUtils.isNotBlank(id) ? getSCIMRoleV2URL() + SCIMCommonConstants.URL_SEPERATOR + id : null;
     }
 
+    /**
+     * Returns the latest SCIM Role URL based on the configuration.
+     * If the configuration USE_ROLE_V2_EP_IN_REF is set to true, it returns the SCIM Role V2 URL.
```

Comment:
update the comment with new configuration

Reply:
d60866075e66319c96a2a6f4689fb851c47118db